feature/repair/src/main/res/values/strings.xml

```diff
@@ -17,6 +17,7 @@
     <string name="support_error_description">Required. Not empty and at least a brief description.</string>
     <string name="text_video">Upload video</string>
```

Comment:
Please be more descriptive with the string name like adding the screen name to it and the action, for example: `repair_send_button`

Reply:
Done! :)